doc/source_tech_preview/deployment/overview.rst

```diff
@@ -20,13 +21,13 @@ SUSE Enterprise Storage.
      default_fontsize = 11;
      deployer [label="Setup deployer"]
      ses_integration [label="SES Integration"]
-     configure [label="Configure\n Cloud"]
+     configure_soc [label="Configure\n Cloud"]
```

Comment:
we already moved the rest from configure_soc to configure in the section below on the branch.